paasta_tools/monitoring/check_capacity.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import argparse
+import json
+import sys
+
+from paasta_tools.api.client import get_paasta_api_client
+from paasta_tools.utils import load_system_paasta_config
+from paasta_tools.utils import paasta_print
+
+
+def parse_capacity_check_options():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--cpus', dest='cpus_default', type=float,
+        help='Default maximum cpu capacity before critical.',
```

Comment:
Can you add a default to all of these?

And then can you try adding the 
https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter
helper so it just spits out the defaults to all these things?